devices.php

```diff
@@ -910,6 +910,9 @@ function buildCustomAttributes($template, $device) {
 					print "\n\t<option value=\"$dcaValue\"$selected>$dcaValue</option>";
 				}
 				echo '</select></div>';
+			} else if ($cvtype=="url") {
+				echo '<div><input type="text"',$validation,' name="',$inputname,'" id="',$inputname,'" value="',$customdata["value"],'">';
+				echo '<button type="button" onclick=window.location.href="',$customdata["value"],'" value="open">Open</button></div>';

```

Comment:
Why do this as a separate if statement instead of just an include for the button portion?  this is duplicating the input line as well which seems like a waste of space.  Second the value of the button hasn't been encoded to be translated correctly.